src/fsharp/FSharp.Core/map.fs

```diff
@@ -933,3 +937,11 @@ module Map =
     [<CompiledName("Count")>]
     let count (table: Map<_, _>) =
         table.Count
+
+    [<CompiledName("Keys")>]
+    let keys (table: Map<_, _>) = 
+        seq { for kvp in table -> kvp.Key }
+
+    [<CompiledName("Values")>]
+    let values (table: Map<_, _>) = 
+        seq { for kvp in table -> kvp.Value }
```

Comment:
Use table.Values.  Note this will include duplicate values

Reply:
This should be fixed now. I added the note of the duplicate values to the xml doc.

Reply:
So I reverted the change because the Map.Values on the type clashed with an internal implementation in illib.fs. I left Map.values in the module with the only implementation.